src/redux/books/book.js

```diff
@@ -0,0 +1,41 @@
+const BOOK_ADDED = 'BOOK_ADDED';
+const BOOK_REMOVED = 'BOOK_REMOVED';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`, kindly make sure to use the `ducks pattern` in naming your action types